src/classes/Carts/Cart.ts

```diff
@@ -126,10 +126,34 @@ export default abstract class Cart {
         return this.our[sku] !== undefined ? this.our[sku].amount : 0;
     }
 
+    getOurGenericCount(sku: string): number {
+        return this.getGenericCount(sku, s => this.getOurCount(s));
+    }
+
+    getGenericCount(sku: string, getCountFn: (sku: string) => number): number {
+        const pSku = SKU.fromString(sku);
+        if (pSku.quality === 5) {
+            // try to count all unusual types
+            return this.bot.unusualEffects
+                .map(e => {
+                    pSku.effect = e.id;
+                    const s = SKU.fromObject(pSku);
+                    return getCountFn(s);
+                })
+                .reduce((p, c) => (p ? p + c : c));
```

Comment:
Are we able to make these more descriptive for future additions?

Reply:
I updated the code